### PR TITLE
Add script to rename packages from ignition to gazebo

### PR DESCRIPTION
**Pull request by j-rivero**
Still a work in progress. First run on gz-cmake3 https://github.com/gazebo-release/gz-cmake3-release/pull/1

### CODE REVIEW

File: release-repo-scripts/rename_ignition_to_gazebo.bash [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  echo "Renaming ${f} -> ${f/ignition-/gz-}"
+  git mv "${f}" "${f/ignition-/gz-}"
+done
+
+# 6. watch files
+find . -name 'watch' -type f | while IFS= read -r f; do
+  echo "Changeing watch file ${f}"
+  sed -i -e 's:/osrf/:/ignitionrobotics/:g' "${f}"
+  sed -i -e 's:ignitionrobotics:gazebosim:g' "${f}"
+  sed -i -e 's:ignition:gz:g' "${f}"
```
Comment (chapulina) [inline release-repo-scripts/rename_ignition_to_gazebo.bash:109]: I think we also need a `sed -i -e 's:ign:gz:g' "${f}"` here, to fix URLs like this:

https://github.com/gazebo-release/gz-cmake3-release/pull/1/files#diff-2fb1332330612d0a6f718a12ca158ab98f3ee801f5e243cc8dd6ceca08cb181aR3

**Review by chapulina — COMMENTED** [review]
I just tried installing CMake3 and it looks like it did the right thing!

**Comment by chapulina** [comment]
This script is very handy! A lot has changed in the release repos since this PR was opened. So I've been using a modified version of it to open various PRs. 

* https://github.com/gazebo-tooling/release-tools/compare/rename_ign_to_gz...chapulina/rename_ign_to_gz

**Comment by j-rivero** [comment]
> This script is very handy! A lot has changed in the release repos since this PR was opened. So I've been using a modified version of it to open various PRs.

Imported changes in [a32c867](https://github.com/gazebo-tooling/release-tools/pull/726/commits/a32c867fe45c7b86af5602a7c35ee37aa08abca6)

**Comment by j-rivero** [comment]
Merging to archive the work done. The script probably will need some tweaks if used in the future but I think it is a nice to have.

**Pull request merged**